specs/FamilySafety.md

```diff
@@ -0,0 +1,114 @@
+Family Safety
+===
+
+# Background
+Our customers have asked for a new API to toggle the Family Safety feature on and off. The Family 
+Safety feature is enabled or disabled for a browser process instance, and the end developer 
+won't be able to change it while the webview2 browser process instance is running. Once enabled, 
+it will provide the same functionally as the browser Family Safety feature including: Activity 
+reports, search filtering, and web filtering. Besides toggling the feature, we also provide an 
+API to add desired sites to the allowed uris, so app functionality will not be impacted by 
+Family Safety. Additionally, parents can also override a block.
+Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void CreateEnvrionmentWithOption()
+{
+    // If parents set filtering rules to only allowed sites. App developers can use
+    // FamilySafetyAllowedUris to show app content and still honor general filter settings set
+    // by parents.
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.IsFamilySafetyEnabled = true;
+    // appassets.example is used as an example app content. All subdomain are added to the allow list
+    options.FamilySafetyAllowedUris.Add("https://appassets.example/*");
+    auto environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+}
+```
+## Win32 C++
+```cpp
+HRESULT InitializeWebView()
+{
+    // If parents set filtering rules to only allowed sites. App developers can use
+    // FamilySafetyAllowedUris to show app content and still honor general filter settings set
+    // by parents.
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    options->put_IsFamilySafetyEnabled(TRUE);
+    // appassets.example is used as an example app content. All subdomain are added to the allow list
+    const WCHAR* allowedUris[1] = {L"https://appassets.example/*"};
+    options->SetFamilySafetyAllowedUris(1, allowedUris);
```

Comment:
When family safety is enabled, isn't there a list of sites to allow/block in the family account already? Is Get returning that cloud information, or is this a local-override that's merged with the cloud values?

Reply:
My interpretation is that this list is merged with the cloud-controlled values. Maybe call this FamilySafetyAdditionalAllowedUris? The word "Additional" already is in use in this class for a similar concept (AdditionalBrowserArguments).

Reply:
Local only. There is no developer access to the family safety's cloud list. Parent explicit block list wins over this local allow list.

Reply:
Let's rename: FamilySafetyAdditionalAllowedUris